controller-client/pom.xml

```diff
@@ -36,12 +36,7 @@
 
     <name>WildFly: Controller Client</name>
 
-    <properties>
-        <!-- maven-compiler-plugin -->
-        <!-- Keep the management client libs on JDK 7 -->
-        <maven.compiler.target>1.7</maven.compiler.target>
-        <maven.compiler.source>1.7</maven.compiler.source>
-    </properties>
+    <properties/>
```

Comment:
I think this was left at 1.7 for some reason, but also believe I have seen some discussion about this